### PR TITLE
PIM-7450: Add styles for remove asset confirmation

### PR DESCRIPTION
**Pull request by tamarasaurus**
**Description (for Contributor and Core Developer)**

This PR adds styles and ACL for removing product asset in asset collection confirmation

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | OK
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/config/acl.yml [hunk 1]
```diff
@@ -76,6 +76,10 @@ pim_enrich_product_history:
     type: action
     label: pim_enrich.acl.product.history
     group_name: pim_enrich.acl_group.product
+pim_enrich_product_asset_removal:
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/config/acl.yml]: This should not be in CE as asset is in EE

File: src/Pim/Bundle/EnrichBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -185,6 +185,7 @@ pim_enrich:
             comment: Comment products
             download: Download the product as PDF
             history: View product history
+            asset_removal: Remove a product asset from an asset collection
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/messages.en.yml]: Same

File: src/Pim/Bundle/UIBundle/Resources/public/less/components/product-edit-form/AssetPreview.less [hunk 3]
```diff
@@ -50,6 +50,29 @@
...
+    box-shadow: 0px 0px 4px rgba(0,0,0,0.2);
+    left: -78px;
+
+    &-hide {
+      display: none;
+    }
+  }
+
+  &-confirmationText {
+    font-size: 16px;
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/less/components/product-edit-form/AssetPreview.less]: Please use only the available font size on variables.less

File: src/Pim/Bundle/EnrichBundle/Resources/config/acl.yml (hunk 1 above)
Comment (tamarasaurus) [reply src/Pim/Bundle/EnrichBundle/Resources/config/acl.yml]: Fixed!

**Pull request merged**